### DISCUSSION TITLE
聊一下用户体验

### DISCUSSION THREAD

**Discussion by DayuGuo** [travellings-link/travellings#1380] [发展建议]
今天申请通过了，第 701 位加入开往，我的博客：https://anotherdayu.com/。
想要试试看开几次能看到自己，就有了这次记录，比较悠闲的在点，并不严谨和精确。

今天共开了 100 次；
第 8 次开到了「开往」的主页；
开失败 11 次，其中 404 三次，其他的未找到「开往」标识；
标志是折叠的或仅在底端，大概有11个；
5 次左右重复的访问；
未找到特别感兴趣的 Blog。

感觉开往的社群和十年之约挺不一样，
开往，比较年轻，程序员偏多，很少看到其他类别的。
十年之约的虫洞，成员写的内容类型更丰富一些。

祝好

**Comment by DayuGuo** [comment]
未开到自己的主页，要出门了，改天再试试。

**Comment by linlinzzo** [comment]
你的网站因为缓存还没有在JS当中，所以跳转到的概率为0。

**Comment by linlinzzo** [comment]
如果遇到网站有问题的，请通过反馈链接跟我们说。或者我的联系方式在下面，可以直接说，我们会及时处理。

QQ： 812568734
微信：xx15759858675

**Comment by linlinzzo** [comment]
然后对开往有什么意见也可以直接跟我们讲。

**Comment by linlinzzo** [comment]
问题解决了，是cdn那边忘记把源站迁移的JS删除掉。现在有概率刷到你的博客了。
